.github/workflows/ci.yaml

```diff
@@ -8,21 +8,24 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9, '3.10']
+        python-version: [3.7, 3.8, 3.9, '3.10', '3.11']
 
     steps:
       - uses: actions/checkout@v2
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
+          channels: conda-forge, defaults
```

Comment:
I don't think we need conda-forge here since this is a different action. I tested it yesterday (but didn't open a PR because it failed on windows)

Reply:
wdym? The ci is passing

Reply:
It doesn't accept the channels like you're saying.

Reply:
yeah, and it'll pass if we remove `channels: conda-forge, defaults`

Reply:
> failed on windows

Wouldn't it keep failing?

Reply:
I just reverted

Reply:
let me open a PR